### PR TITLE
Use normal font weight for input field

### PR DESCRIPTION
**Pull request by Kamielvf**
image

Fixes the issue above, also looks better :+1:

### CODE REVIEW

**Comment by SomeoneWithAnInternetConnection** [comment]
What's it look like *after* the fix?

**Comment by Kamielvf** [comment]
image

File: runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ public HiscorePanel()
 
 		input = new IconTextField();
 		input.setIcon(search);
-		input.setFont(labelFont.deriveFont(Font.BOLD));
+		input.setFont(labelFont);
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePanel.java]: You can just remove the labelFont completely here, as the default one is exactly same, previous logic was just making it bold that is why it is here.

**Comment by deathbeam** [comment]
Any update on the requested changes?

**Comment by Kamielvf** [comment]
:+1:

**Pull request merged**